### PR TITLE
Pbi 393523 gs ingress

### PR DESCRIPTION
**Pull request by jasnames**
The dev guide changes are only to fix a few formatting issues.

### CODE REVIEW

File: content/developer-guide/omf-dev/omf-ingress-specification.md [hunk 1]
```diff
@@ -4,59 +4,49 @@ uid: omfIngressSpecification
 
 # Use Open Message Format with AVEVA Data Hub
 
-[The Open Message Format (OMF) specification](https://docs.osisoft.com/bundle/omf/) is generic in that it does not specify a particular back-end system. This topic is a companion to the OMF specification which describes how OMF is interpreted by AVEVA Data Hub back-end system. 
+[The Open Message Format (OMF) specification](https://docs.osisoft.com/bundle/omf/) is generic in that it does not specify a particular back-end system. This topic is a companion to the OMF specification which describes how OMF is interpreted by the AVEVA Data Hub back-end system.
```
Comment (osisoft-mbishop) [inline content/developer-guide/omf-dev/omf-ingress-specification.md:7]:
```suggestion
[The Open Message Format (OMF) specification](https://docs.osisoft.com/bundle/omf/) is generic in that it does not specify a particular back-end system. This topic is a companion to the OMF specification, which describes how OMF is interpreted by the AVEVA Data Hub back-end system. 
```

File: content/developer-guide/omf-dev/omf-ingress-specification.md [hunk 2]
```diff
@@ -4,59 +4,49 @@ uid: omfIngressSpecification
...
 
 ### Type messages
-  A *type* message is interpreted by AVEVA Data Hub as an SdsType in the Sequential Data Store. 
-  Because SdsTypes are immutable, update operations are not supported. The keywords in the 
-  Type definition are interpreted as follows:
+A *type* message is interpreted by AVEVA Data Hub as an SdsType in the Sequential Data Store. Because SdsTypes are immutable, update operations are not supported. The keywords in the Type definition are interpreted as follows:
 
-  + ``id``: Corresponds to the SdsType Id field. It must conform to the rules defined for a 
+- ``id``: Corresponds to the SdsType Id field. It must conform to the rules defined for a 
     typeId specified here: [Types](xref:sdsTypes)
```
Comment (osisoft-mbishop) [inline content/developer-guide/omf-dev/omf-ingress-specification.md:21]:
```suggestion
    typeId specified here: [Types](xref:sdsTypes).
```

Comment (osisoft-mbishop) [inline content/developer-guide/omf-dev/omf-ingress-specification.md:21]:
```suggestion
    typeId specified here: [Types](xref:sdsTypes).
```

File: content/developer-guide/omf-dev/omf-ingress-specification.md [hunk 3]
```diff
@@ -4,59 +4,49 @@ uid: omfIngressSpecification
...
 
-### Property types and formats
-  OMF supports setting the ``format`` keyword to specify how a particular JSON type should 
-  be interpreted. The following is a mapping for the Sequential Data Store supported 
-  types (see [types](xref:sdsTypes)):
+- ``tags``: Currently unsupported.
+
+- ``metadata``: Currently unsupported.
+
+The ``isindex`` keyword corresponds to the ``isKey`` attribute of an SdsTypeProperty. SdsTypes support clustered indexes which can be specified with multiple properties marked with the ``isindex`` keyword with a value of ``true``. For compound indexes, the index property order within the message corresponds to the ``Order`` field of an SdsTypeProperty. The ``isname`` keyword is not supported.
```
Comment (osisoft-mbishop) [inline content/developer-guide/omf-dev/omf-ingress-specification.md:37]:
```suggestion
The ``isindex`` keyword corresponds to the ``isKey`` attribute of an SdsTypeProperty. SdsTypes support clustered indexes, which can be specified with multiple properties marked with the ``isindex`` keyword with a value of ``true``. For compound indexes, the index property order within the message corresponds to the ``Order`` field of an SdsTypeProperty. The ``isname`` keyword is not supported.
```

File: content/overview/get-started/gs-ingress.md [hunk 4]
```diff
@@ -4,24 +4,32 @@ uid: gs-ingress
...
+
+1. <xref:gpClientCredentialsClient>. The edge device uses this client to authenticate with AVEVA Data Hub.
+
+1. <xref:gpOMFConnection>. Connect your edge devices to AVEVA Data Hub.
+
+## Option C: Ingress data programmatically
+
+APIs provide programmatic access to bring sequential data into AVEVA Data Hub. This can be done through custom OMF applications or utilizing SDS data writes.
+
+See <xref:omfVsSds>.
```
Comment (osisoft-mbishop) [inline content/overview/get-started/gs-ingress.md:35]: It seems like we should link to content/developer-guide/sequential-data-store-dev/sds-write-data.md here and then make the omfVsSds topic a note.

**Review by osisoft-mbishop — CHANGES_REQUESTED** [review]
I have couple of grammar nitpicks on stuff you didn't write, but for the gs-ingress topic, I think for option C we should link to a different topic. See comments.

**Comment by jasnames** [comment]
> 

I didn't expect you to read the Dev sections. :-) I updated Option C. Thanks!

**Pull request merged**